src/app/query/bitcoin/runes/runes-outputs-by-address.query.ts

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from '@tanstack/react-query';
+
+import type { AppUseQueryConfig } from '@app/query/query-config';
+import { useBitcoinClient } from '@app/store/common/api-clients.hooks';
+
+import type { RunesOutputsByAddress } from '../bitcoin-client';
+
+export function useGetRunesOutputsByAddressQuery<T extends unknown = RunesOutputsByAddress[]>(
+  address: string,
+  options?: AppUseQueryConfig<RunesOutputsByAddress[], T>
+) {
+  const client = useBitcoinClient();
+
+  return useQuery({
+    queryKey: ['runes-outputs-by-address', address],
```

Comment:
If you ref my code you can remove the hardcoded testnet path and use the `runesEnabled` flag so we can flip mainnet on when we need to.